models/person.go

```diff
@@ -86,3 +83,12 @@ func (c *Person) SetCountriesInfo(results *sparql.Results) {
 		c.Countries = append(c.Countries, country)
 	}
 }
+
+// SetPersonSubtypesInfo adds subtypes to a person from sparql results
+func (c *Person) SetPersonSubtypesInfo(results *sparql.Results) {
+	solutions := results.Solutions()
+	for _, solution := range solutions {
+		subtype := &Labeled{solution["subtype"].String(), solution["subtype"].String()[31:]}
```

Comment:
This doesn't seem obvious, so can you leave a comment about why you're taking a slice from the end of this string.